### PR TITLE
Simplified demo

### PR DESCRIPTION
**Pull request by eric978**
Kind.sh now modified to use kubeslice-cli.    Otherwise, functionality should stay the same.

### CODE REVIEW

**Comment by jhveras-avesha** [comment]
Something seems to be wrong with worker-2:

image

I'll take a closer look at it tomorrow

File: kind/config/topology.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      context_name: kind-controller
+    workers:
+      - name: worker-1
+        context_name: kind-worker-1
+      - name: worker-2
+        context_name: kind-worker-2      
+  kubeslice_configuration:
+    project_name: demo
+  helm_chart_configuration:
+    repo_alias: kubeslice
```
Comment (richiesebastian) [inline kind/config/topology.yaml:15]: It is using old charts repo https://kubeslice.github.io/charts/ it should point to https://kubeslice.github.io/kubeslice/

Comment (eric978) [reply kind/config/topology.yaml:15]: Ok, fixed

**Comment by jhveras-avesha** [comment]
Tried again and I'm still having issues with kind.sh. Found this in the logs of nsmd container:

```
time="2022-10-12T06:10:13Z" level=info msg="Requesting nsmd for client connection..."
time="2022-10-12T06:10:13Z" level=info msg="NSM gRPC socket: /var/lib/networkservicemesh/nsm.io.sock is operational" operation=nsm-server-start span="57e0dea48e6ab841:2469768b0c49ab91:57e0dea48e6ab841:1"
time="2022-10-12T06:10:13Z" level=info msg="create monitor servers" operation=nsm-server-start span="57e0dea48e6ab841:2469768b0c49ab91:57e0dea48e6ab841:1"
time="2022-10-12T06:10:13Z" level=info msg="CrossConnect - Serve starting..."
time="2022-10-12T06:10:13Z" level=info msg="RemoteConnection - Serve starting..."
2022/10/12 06:10:13 too many open files
```
Also attached kind.sh output, just in case.
[kind.log](https://github.com/kubeslice/examples/files/9762070/kind.log)

**Comment by jhveras-avesha** [comment]
I've just applied this solution from kind doc: https://kind.sigs.k8s.io/docs/user/known-issues/#pod-errors-due-to-too-many-open-files but I'm still getting this from worker-1 (controller, manager pod):

`Events:
  Type     Reason     Age                     From               Message
  ----     ------     ----                    ----               -------
  Normal   Scheduled  7m49s                   default-scheduler  Successfully assigned kubeslice-system/kubeslice-operator-5d979fffcd-97jgx to worker-1-worker
  Normal   Pulling    7m43s                   kubelet            Pulling image "gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0"
  Normal   Pulled     7m8s                    kubelet            Successfully pulled image "gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0" in 35.472260214s
  Normal   Created    7m8s                    kubelet            Created container kube-rbac-proxy
  Normal   Started    7m8s                    kubelet            Started container kube-rbac-proxy
  Normal   Pulling    7m8s                    kubelet            Pulling image "docker.io/aveshasystems/worker-operator:0.11.0"
  Normal   Pulled     6m49s                   kubelet            Successfully pulled image "docker.io/aveshasystems/worker-operator:0.11.0" in 18.341205295s
  Warning  Failed     5m8s (x10 over 6m49s)   kubelet            Error: couldn't find key endpoint in Secret kubeslice-system/kubeslice-hub
  Normal   Pulled     2m43s (x20 over 6m49s)  kubelet            Container image "docker.io/aveshasystems/worker-operator:0.11.0" already present on machine
`

**Review by jhveras-avesha — APPROVED** [review]

**Pull request merged**